.travis.yml

```diff
@@ -1,21 +1,45 @@
 language: node_js
 node_js:
   - '10.15'
+
+addons:
+  apt:
+    packages:
+      # Ubuntu 16+ does not install this dependency by default, so we need to install it ourselves
+      - libgconf-2-4
+    
 cache:
   npm: true
+  directories:
+    # we also need to cache folder with Cypress binary
+    - ~/.cache
 
 env:
   global:
     - REACT_ENV=development
+    - BABEL_ENV=development
     - REACT_APP_LOG_LEVEL=debug
     - BUNDLESIZE_GITHUB_TOKEN=63f6d1717c6652d63234cf9629977b08f4bac3fd
     #    - REACT_APP_NETWORK_ID=4447
     - MNEMONIC="choice congress hobby buddy dutch busy army eager empty solution start grunt"
+    - NETLIFY_AUTH_TOKEN=95afeb17e7327bfcd28ea26a0831e6d5956ae83cfa4de9903500178a3e534024
     - CI=false
-script:
-  - npm run coverage
-  - npm run coveralls
-  - npm run build
+
+matrix:
+  include:
+    - script: 
+      - npm install jq -g
+      - npm install cypress
+      - npm run build
+      - npm install jq -save-dev
+      - npm install cypress --save-dev 
+      - npm install netlify-cli --save-dev
+      - netlifyctl deploy --draft        
```

Comment:
this output of this command should be inserted to an env, just like you did before
URL=netlifyctl deploy --dratf | jq....